libs/pxweb2-ui/src/lib/components/Typography/BodyLong/BodyLong.module.scss

```diff
@@ -0,0 +1,29 @@
+@use "../../../../../style-dictionary/dist/scss/fixed-variables.scss" as fixvar;
```

Comment:
It looks like the file hasn't been formatted by prettier, do you have the extension installed in vscode?